javaslang/src-gen/test/java/javaslang/APITest.java

```diff
@@ -230,6 +230,26 @@ public void shouldLazyReturnNotNull() {
         assertThat(lazy(() -> 1)).isNotNull();
     }
 
+    @Test
+    public void shouldOptionReturnNotNull() {
+        assertThat(option(1)).isNotNull();
```

Comment:
The _semantics_ are checked in OptionTest.

Here we should check if the _aliases_ work right. E.g.

* `assertThat(Option(1)).isEqualTo(Option.of(1));`
* `assertThat(Some(1)).isEqualTo(Option.some(1));`
* `assertThat(None()).isEqualTo(Option.none());`

Bonus points:

* `assertThat(Option(null)).isEqualTo(Option.of(null));

(I used first-upper case methods here because it _might_ be our API, but before changing it let me investigate my idea further of letting value types implement Pattern*)